src/pybind/ceph_daemon.py

```diff
@@ -187,11 +187,11 @@ def format_dimless(self, n, width):
         Format a number without units, so as to fit into `width` characters, substituting
         an appropriate unit suffix.
         """
-        units = [' ', 'k', 'M', 'G', 'T', 'P', 'E', 'Z']
+        units = [' ', 'K', 'M', 'G', 'T', 'P', 'E', 'Z']
```

Comment:
i think "k" is correct. see https://en.wikipedia.org/wiki/Unit_prefix and https://en.wikipedia.org/wiki/Binary_prefix .